ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/design.schema.yml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  design:
+    documentation_options:
+      filename: Fabric Topology
+      table: Design Variables
+    type: dict
+    keys:
+      type:
+        documentation_options:
+          filename: Fabric Topology
+          table: Design Variables
+        type: str
+        description: |
+          By setting the design.type variable, the default node-types and templates described in these documents will be used.
```

Comment:
Which documents? ;)

Reply:
All descriptions will be reviewed in the next rounds of PRs where we focus on building the documentation.